html/pfappserver/root/src/views/Configuration/pki/profiles/_components/TheForm.vue

```diff
@@ -119,6 +119,38 @@
           :text="$i18n.t('Cloud Service to integrate.')"
         />
       </base-form-tab>
+      <base-form-tab :title="$i18n.t('Renewal Configuration')">
+        <form-group-days-before-renewal namespace="days_before_renewal"
+          :column-label="$i18n.t('Days before renewal')"
+          :text="$i18n.t('Number of days before the pki authorize renewal. Set to 0 mean renewal is always allowed')"
+        />
+        <form-group-renewal-mail namespace="renewal_mail"
+          :column-label="$i18n.t('Renewal Email')"
+          :text="$i18n.t('Send email to the owner when the certificate is about to expire.')"
+        />
+        <form-group-days-before-renewal-mail namespace="days_before_renewal_mail"
+          :column-label="$i18n.t('Days before sending renewal email')"
+          :text="$i18n.t('Number of days before expiration of the certificate that trigger sending email.')"
```

Comment:
Number of days before expiration of the certificate that trigger sending email. => Number of days before certificate expiration to trigger sending email.